vumi_http_proxy/twistd.pid

```diff
@@ -0,0 +1 @@
+31716
```

Comment:
You should maybe at `twistd.pid` to `.gitignore` (and delete this file from the PR).